src/Raven.Server/Documents/ETL/Providers/Queue/QueueEtl.cs

```diff
@@ -176,6 +182,50 @@ public QueueEtlTestScriptResult RunTest(IEnumerable<QueueWithItems<T>> records,
             Summary = summaries
         };
     }
+    
+    protected sealed class CloudEventConverter : JsonConverter<CloudEvent>
```

Comment:
Q: Is this file (`QueueEtl`) a valid place for this class?

Reply:
Let's move it to a separate file